react/__test__/__snapshots__/index.test.tsx.snap

```diff
@@ -366,6 +366,41 @@ exports[`Rich text component should render list 1`] = `
 </DocumentFragment>
 `;
 
+exports[`Rich text component should render ordered list 1`] = `
+<DocumentFragment>
+  <div
+    class="container flex tl items-start justify-start t-body c-on-base"
+  >
+    <div>
+      <p
+        class="lh-copy paragraph"
+      >
+        Teste 
```

Comment:
```suggestion
        Test
```

Reply:
I'm keeping the tests similar to the previous which is written like that.